addons/mail/static/src/messaging/entity/activity/activity.js

```diff
@@ -16,15 +16,13 @@ function ActivityFactory({ Entity }) {
         /**
          * @override
          */
-        delete() {
-            if (!this._isRecordDeleted) {
-                this.env.rpc({
-                    model: 'mail.activity',
-                    method: 'unlink',
-                    args: [[this.id]],
-                });
-            }
-            super.delete();
+        deleteRecord() {
+            this.env.rpc({
```

Comment:
I think github lost a comment.

It said this needs await.

Reply:
Nop, I remember I read that, just forgot to add it here.. sorry